MBBSEmu.Tests/ExportedModules/Majorbbs/outprf_Tests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class outprf_Tests : ExportedModuleTestBase
+    {
+        private const int OUTPRF_ORDINAL = 463;
+
+        [Fact]
+        public void outprf_rtkick_Test()
```

Comment:
i'm not entirely sure how this relates to rtkick (at least in the test), maybe just outprf_noChannel_Test()?

Reply:
rtkick passes in channel of `-1`, which will never be a channel number. This test I labeled `rtkick` only because of the `-1` value vs. just a random channel. It should renamed to just a test for non-existent channel.